src/formula.py

```diff
@@ -154,8 +154,12 @@ class UnixPackageFormula(Formula):
     install_to: str | None = None
 
     def install(self) -> None:
+        if self.install_to is not None:
```

Comment:
You could also set `install_to: str = '/usr/local` on the class level to give it that default value.

Reply:
done!